core/modules/qdisp/JobQuery.cc

```diff
@@ -102,7 +102,7 @@ bool JobQuery::runJob() {
         // only if the executive has not already been cancelled. The cancellation
         // procedure changes significantly once the executive calls xrootd's Provision().
         bool success = executive->xrdSsiProvision(_queryResourcePtr, qr);
```

Comment:
Is this really needed instead of just:

  // The only way this can fail is if the user query is cancelled.
  if (executive->xrdSsiProvision(_queryResourcePtr, qr)) return true;

First of all you have fewer lines of code (more observable code).
Secondly you don't need to repeat the method name within the comment (fewer things to take care of should you be doing code refactoring)